views/partials/form.handlebars

```diff
@@ -1 +1,49 @@
-Edit topic form
\ No newline at end of file
+<!-- Edit topic form -->
+<form>
+    <div class="row">
+            <div class="col-lg-4">
+                <div class="form-group">
+                    <input type="text" placeholder="subjet name" class="form-control">
```

Comment:
let's add the `name` attribute back on all these inputs, so the form posts the proper request

`<input type="text" name='subjectName' placeholder="subject name" class="form-control">`

..and we also need to add the proper name attributes on all the inputs below

please look at your mongoose model for all the names that need to be added